projects/canopy/src/lib/pagination/docs/guide.stories.mdx

```diff
@@ -0,0 +1,100 @@
+
+...
+
+export class PagingExampleComponent implements OnInit {
+  allItems: = [];
+  pagedItems: [];
+  itemsPerPage = 10;
+
+  ngOnInit(): void {
+    this.allItems = this.someService.getData();
+
+    // Create the first page of data on init
+    this.pagedItems = this.getPagedData(0, this.itemsPerPage - 1);
+  }
+
+  onPageChanged(pageData: PageData): void {
+    // And then respond to onPageChanged events to update the paged data
+    this.pagedItems = this.getPagedData(pageData.startIndex, pageData.endIndex)
+  }
+
+  private getPagedData(startIndex: number, endIndex: number): unknown[] {
+    // Important! Be careful if using array.prototype.splice as it returns
+    // items from startIndex, up to but not including endIndex.
+    return this.allItems.slice(startIndex, endIndex + 1);
+  }
+}
+```
+
+### Inputs
+
+| Name           | Description                                                                        | Type      | Default                           | Required |
+|----------------|------------------------------------------------------------------------------------|-----------|-----------------------------------|----------|
+| `id`           | HTML ID attribute, auto generated if not provided                                  | `string`  | `lg-pagination-${++nextUniqueId}` | No       |
+| `totalItems`   | The total number of items in the array being paged                                 | `number`  | `0`                               | No       |
+| `itemsPerPage` | The number of items to display per page                                            | `number`  | `10`                              | No       |
+| `currentPage`  | The current page, Can be programatically set but is usually controlled internally  | `number ` | `1`                               | No       |
+
+### Outputs
+
+| Name   | Description                             | Type                  | Default | Required |
+|--------|-----------------------------------------|-----------------------|---------|----------|
+| `pagedChanged` | Event emitted whenever the page needs to be changed because either an input property has changed or the user has clicked on one of the page buttons. This event can also be used to set the focus to the updated page section | `EventEmitter<PageData>` | n/a     | No       |
+
+
+```typescript
+export interface PageData {
+  // The requested page number.
+  page: number;
+
+  // The requested zero based start index of the data being paged.
+  startIndex: number;
+
+  // The requested end index (inclusive) of the data being paged
+  endIndex: number;
+}
+```
+
+## Additional notes
+The component will automatically hide itself when the the total number of items is **less than or equal to** the `itemsPerPage` input (i.e. There is only 1 page of data).
+
+The label (Showing x-y of x results) is positioned automatically, either to the right or below the pagination buttons, depending on the available space
```

Comment:
Missing full stop at the end of this sentence

Reply:
done